package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "webpack",
+  "version": "1.0.0",
+  "description": "> Js Group Project made by a group of two teams",
+  "private": true,
+  "scripts": {
+    "test": "jest --env=jsdom",
+    "start": "webpack serve --open",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/yayner2002/webpack.git"
+  },
```

Comment:
- [ ] Kindly change the `url` to the accurate URL. Kindly implement this change. 👍